rdkcentral/Lightning-UI-Components
# fix(Metadata): only create text elements when needed

## Description



MetadataBase instantiates Textbox components for Title, Description, and Details, and an Icon component for the Logo regardless of whether or not these properties are set. Tiles often don't need all of these elements. This change waits to create them until they're needed. 

## Testing



- Ensure MetadataBase and MetadataTile render properly. 
- Set up a Row or Column of Tiles with with Text/Description properties that will require Marquee. Ensure marquee behavior is correct as you navigate across the row. 

## Checklist
